Gamify-IT/test-data
# Add test data for Keycloak

Part of Gamify-IT/issues#193

## How to test this

1. Clone <https://github.com/Gamify-IT/landing-page>
2. Use `docker-compose up` (the docker-compose is currently configured to use the test-data from this branch)
3. Go to <http://localhost:80>
4. Check if you can log in with user `student` password `student`
